app/graphql/cache.ts

```diff
@@ -19,8 +21,8 @@ gql`
     }
   }
 
-  query realtimePrice {
-    realtimePrice {
+  query realtimePrice($currency: DisplayCurrency!) {
+    realtimePrice(currency: $currency) {
```

Comment:
Will we ever use a price feed without passing in a display currency? If not, we should talk to the backend about whether or not even allowing the query to not take a display currency should be a possibillity. The alternative to passing in the display currency, is using the pricefeed as our source of truth for the display currency rather than using the display currency query.

Reply:
Price feed is default USD in the backend. Can use useful for galoy-pay for instance.

For the mobile I don't think we want to use the default, we always want to pass a value